authui/src/authflowv2/components/secondary-button.css

```diff
@@ -11,7 +11,7 @@
     );
     --secondary-btn__font-weight: var(--typography-label-large__font-weight);
     --secondary-btn__ring-width: 1.5px;
-    --secondary-btn__ring-color: var(--color-grey-100);
+    --secondary-btn__ring-color: #f0f1f1;
```

Comment:
Should use grey 100 here.
<img width="614" alt="Screenshot 2024-01-24 at 11 47 12 AM" src="https://github.com/authgear/authgear-server/assets/34942210/a1d455a1-c10c-49d9-949e-8c7556a02fae">

Reply:
Looks like the alternative login button's have different style then, confirming with designer.

Reply:
I think designer pointed it to the incorrect grey-100 in the dark mode palette. We can keep using the original grey-100 first.